src/ComputeGeometryAlgorithmDriver.C

```diff
@@ -135,7 +136,106 @@ ComputeGeometryAlgorithmDriver::post_work()
     realm_.periodic_field_update(dualNodalVolume, fieldSize);
   }
 
+  if ( realm_.checkJacobians_ ) {
+    check_jacobians();
```

Comment:
Right now this step will be executed at every timestep when there is mesh motion, but no mesh deformation or percept refinement. Should we have a flag disabling this check after the initialization phase?

Reply:
Yes, my intent was to always check when mesh motion is active. I can probably add an extra logic statement on mesh deformation. Sadly, this would include some sort of flag management.